plugin/src/main/java/net/dzikoysk/funnyguilds/concurrency/requests/war/WarInfoRequest.java

```diff
@@ -14,19 +13,22 @@
 import net.dzikoysk.funnyguilds.nms.heart.GuildEntityHelper;
 import net.dzikoysk.funnyguilds.shared.bukkit.ChatUtils;
 import net.dzikoysk.funnyguilds.user.User;
+import org.bukkit.Server;
 import org.bukkit.entity.Player;
-import panda.std.Option;
 
 public class WarInfoRequest extends DefaultConcurrencyRequest {
 
     private InfoCommand infoExecutor;
 
     private final GuildEntityHelper guildEntityHelper;
 
+    private final Server server;
     private final User user;
     private final int entityId;
 
     public WarInfoRequest(FunnyGuilds plugin, GuildEntityHelper guildEntityHelper, User user, int entityId) {
+        this.user = user;
+        this.server = plugin.getServer();
```

Comment:
Skoro i tak ta klasa wymaga pluginu, to po co wyciagać z niego elementy tutaj. Trzymaj plugin i wyciągaj z niego potem.